docs/GettingStartedDocs/Contributors/RunningStaticAnalysis.md

```diff
@@ -40,3 +40,29 @@
     ```
 
     Running `scan-view /tmp/scan-build-2019-12-12-102130-18694-1` will open results in a web browser.
+
+## Interpreting warnings
+
+Ideally, all warnings would be legitimate and CI/CD could reject PRs which introduce scan-build errors.
+Unfortunately there are currently many scan-build errors in the OE codebase, and many of them are in
+external code.
+
+Specifically, errors reported from the 3rdparty directory should be ignored. Any other warnings are considered
+bugs and should be fixed or marked as false-positives.
```

Comment:
Can you be more specific on how they are "marked as false-positives"?

Reply:
I decided to remove this piece. Technically an ifdef can be added to keep the code from being analyzed, but I have yet to find a case where this is necessary. It is almost always correct to actually fix the issue and I don't want to encourage band-aid fixes by putting them in the documentation.